util/container/Dockerfile

```diff
@@ -46,6 +46,13 @@ RUN OBS_URL="https://download.opensuse.org/repositories"; \
     mv "$TMPDIR/obs.asc"  /etc/apt/trusted.gpg.d/obs.asc && \
     mv "$TMPDIR/obs.list" /etc/apt/sources.list.d/edatools.list
 
+# Add bazel repository to apt sources.
+RUN apt-get install -y apt-transport-https curl gnupg \
+    && curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg \
+    && mv bazel.gpg /etc/apt/trusted.gpg.d/ \
+    && echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list \
```

Comment:
It looks like this line is breaking in the `build_docker_containers` job. I think Docker doesn't require running with this with `sudo`.

Reply:
Thanks looks like I missed that one.